01-js-intro-and-jquery/js/javascript.js

```diff
@@ -0,0 +1,84 @@
+
+
+var functionFadeToggleSection = (function() {
+  //Private variables
+  var selectedSection = $('section.hidden-section');
```

Comment:
You can target desired elements just with the class (``.hidden-section``), no need to add the element before. :)